src/client/utils/dom.js

```diff
@@ -2,7 +2,8 @@ import CONST from '../../const';
 import SHADOW_UI_CLASSNAME from '../../shadow-ui/class-name';
 import trim from '../../utils/string-trim';
 import nativeMethods from '../sandbox/native-methods';
-import urlUtils from '../utils/url';
+import sharedUrlUtils from '../../utils/url';
```

Comment:
I was hoping we will not have this `shareUrlUtils` NEVER EVER EVER again. Please, don't do that =)